### PR TITLE
V2 Updated test helper for Partial View

### PR DESCRIPTION
**Pull request by nhudinh0309**
This PR focuses on updating the test helper to fixing the failed tests of Partial View and adding the tests for Query Builder in Partial View. It includes:

- Updated api test helper for Partial View as the endpoints are changed
- Updated ui test helper for Partial View as the UI's changes
- Updated UiBaseLocator to add method for adding query builder and rename
- Bumped version

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@umbraco/playwright-testhelpers",
-  "version": "2.0.0-beta.12",
+  "version": "2.0.0-beta.13",
```
Comment (andr317c) [inline package.json:3]:
```suggestion
  "version": "2.0.0-beta.14",
```

There is already a version 2.0.0-beta.13 released

File: lib/helpers/PartialViewApiHelper.ts [hunk 2]
```diff
@@ -8,45 +8,55 @@ export class PartialViewApiHelper {
...
   }
 
-  async update(partialView) {
-    const partialViewData = {
-      "name": partialView.name,
-      "content": partialView.content,
-      "existingPath": partialView.path
-    };
-    return await this.api.put(this.api.baseUrl + '/umbraco/management/api/v1/partial-view', partialViewData);
+  async update(path: string, content: string) {
```
Comment (andr317c) [inline lib/helpers/PartialViewApiHelper.ts:33]: I think the naming for updating content should be updateContent()
And updating name should be updateName()

File: lib/helpers/UiBaseLocators.ts [hunk 3]
```diff
@@ -108,9 +132,10 @@ export class UiBaseLocators {
     await this.page.waitForTimeout(1000);
     await this.modalCaretBtn.click({force: true});
     await this.page.getByLabel(dictionaryName).click();
-    await this.submitBtn.click();
+    await this.chooseDictionaryBtn.click();
   }
 
+  // TODO: replace this method by addQueryBuilderWithOrderByStatement() in all tests and remove it
```
Comment (andr317c) [inline lib/helpers/UiBaseLocators.ts:138]:
```suggestion
  // TODO: replace this method by addQueryBuilderWithOrderByStatement() in all tests and remove it
```

File: lib/helpers/PartialViewUiHelper.ts [hunk 4]
```diff
@@ -33,13 +35,15 @@ export class PartialViewUiHelper extends UiBaseLocators{
     await this.newPartialViewFromSnippetBtn.click();
   }
 
+  // TODO: this method should place in UiBaseLocator as it is used for Stylesheet and Script as well
```
Comment (andr317c) [inline lib/helpers/PartialViewUiHelper.ts:38]:
```suggestion
  // TODO: this method should place in UiBaseLocator as it is used for Stylesheet and Script as well
```

**Review by andr317c — CHANGES_REQUESTED** [review]
Had a few comments

File: lib/helpers/PartialViewApiHelper.ts (hunk 2 above)
Comment (nhudinh0309) [reply lib/helpers/PartialViewApiHelper.ts]: Makes sense

File: lib/helpers/UiBaseLocators.ts (hunk 3 above)
Comment (nhudinh0309) [reply lib/helpers/UiBaseLocators.ts:138]: I will implement it later to avoid the conflict

**Review by andr317c — APPROVED** [review]
LGTM!

**Pull request merged**